test/configCases/records/issue-2991/webpack.config.js

```diff
@@ -0,0 +1,26 @@
+var path = require("path");
+
+module.exports = {
+	entry: "./test",
+	recordsPath: path.resolve(__dirname, "../../../js/config/records/issue-2991/records.json"),
+	target: "node",
+	node: {
+		__dirname: false
+	},
+	plugins: [
+		{
+			apply(compiler) {
+				compiler.plugin("normal-module-factory", (nmf) => {
```

Comment:
Ah ok.

I think you can cause this ignore with a `browser: { foo: false }` in package.json in a browser build. The tests run with target node, so you may add the browser field with `resolve.aliasFields: [ "browser" ]`.